Sources/SwiftParserDiagnostics/MissingNodesError.swift

```diff
@@ -89,7 +96,15 @@ fileprivate enum NodesDescriptionPart {
 
         switch (parts.last, newPart) {
         case (.tokensWithDefaultText(let previousTokens), .tokensWithDefaultText(let newTokens)):
-          parts[parts.count - 1] = .tokensWithDefaultText(previousTokens + newTokens)
+          // Merge `tokensWithDefaultText` if they occur consecutively in the tree
+          if let lastPrevious = previousTokens.last,
+            let firstNew = newTokens.first,
+            originalTokens[lastPrevious, default: lastPrevious].nextToken(viewMode: .all) == originalTokens[firstNew, default: firstNew]
```

Comment:
Don't we want `.sourceAccurate` here?

Reply:
No. We don’t want to merge them if there is a missing token in between them (because the missing token will probably get added by the Fix-It)